scripts/ensure-env.js

```diff
@@ -0,0 +1,7 @@
+var fs = require("fs");
+var env = require("path").join(__dirname, "../", ".env");
+
+fs.exists(env, function (exists) {
```

Comment:
We don't need async here, so keeping things synchronous actually yields a more predictable code path:
```
var exists = fs.existsSync(env);
if (exists) ...
...
```